src/components/Notification/InactivityMessage.jsx

```diff
@@ -1,7 +1,7 @@
 // React Imports
-import React, { useState, useEffect } from 'react';
-// Inrupt Library Imports
-import { LogoutButton } from '@inrupt/solid-ui-react';
+import React, { useState, useEffect, useRef } from 'react';
+// Inrupt Imports
+import { useSession } from '@hooks';
 // Material UI Imports
 import Button from '@mui/material/Button';
 import Dialog from '@mui/material/Dialog';
```

Comment:
MUI imports should be rearranged alphabetically. All other files are alphabetical.